src/features/pages/home/live-pricing/use-live-columns.tsx

```diff
@@ -30,15 +30,25 @@ const useLiveColumns = () => {
                 cell: (info) => (
                     <Flex.Box>
                         {info.row.original.mkt !== 'etfs' ? (
-                            <SymbolIcon icon_src={info.getValue()} />
+                            <SymbolIcon
+                                icon_src={info.getValue()}
+                                alt={info.row._valuesCache.sym as string}
+                            />
                         ) : (
-                            <SymbolIcon icon_src="ETFSICON" />
+                            <SymbolIcon
+                                icon_src="ETFSICON"
+                                alt={info.row._valuesCache.sym as string}
+                            />
                         )}
                     </Flex.Box>
                 ),
             }),
             liveMarketColumnHelper.accessor('sym', {
-                header: () => <Flex.Box></Flex.Box>,
+                header: () => (
+                    <Flex.Box>
+                        <p className="sr-only">Symbol</p>
```

Comment:
Curious about why the p tag is used here instead of `Typography.Paragraph` and why translation is not used here?

Reply:
The p tag was used here because no style was required. And translation is not using because this text is only for screen reader